protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -475,23 +475,23 @@ service TradingDataService {
 
   // List liquidity provisions
   //
-  // Get a list of liquidity provisions for a given market
+  // Get a list of liquidity provisions for a given market.
   rpc ListLiquidityProvisions(ListLiquidityProvisionsRequest) returns (ListLiquidityProvisionsResponse) {
-    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Liquidity provisions"};
```

Comment:
There's also a "liquidity providers" tag that has 1 endpoint in it. I wonder if these two should be combined. https://docs.vega.xyz/testnet/category/api/rest/data-v2/liquidity-providers

Reply:
Good spot, I'll add that in too.